### PR TITLE
Add task feature in dependency on fluvio-future

### PR DESCRIPTION
**Pull request by nacardin**
Update deps

### CODE REVIEW

File: src/k8-client/Cargo.toml [hunk 1]
```diff
@@ -1,50 +1,48 @@
...
+bytes = "1.0.1"
 base64 = { version = "0.13.0" }
-futures-util = { version = "0.3.5", features = ["io"]}
-rand = { version = "0.7.3" }
+futures-util = { version = "0.3.5", features = ["io"] }
+rand = { version = "0.8.3" }
 openssl = { version = "0.10.30", optional = true }
 rustls = { version = "0.19.0", optional = true }
-hyper = { version = "0.13.8", default-features=false, features=["stream"] }
+hyper = { version = "0.14.4", features = ["client", "http1", "http2", "stream"] }
```
Comment (sehz) [inline src/k8-client/Cargo.toml:28]: do we need http2?

**Review by sehz — CHANGES_REQUESTED** [review]
minor question

File: src/k8-client/Cargo.toml (hunk 1 above)
Comment (nacardin) [reply src/k8-client/Cargo.toml:28]: Kubernetes API Server supports http2

Comment (sehz) [reply src/k8-client/Cargo.toml:28]: sounds good

**Review by sehz — APPROVED** [review]

**Pull request merged**